### DISCUSSION TITLE
Wow! Its Awesome <3 | New GitHub Achievements Badges

### DISCUSSION THREAD

**Discussion by atulkamble** [community/community#18270] [Other Feature Feedback, Questions, & Ideas]
**GitHub Achievements** is Awesome. Just gone through the [article: Introducing Achievements: recognizing the many stages of a developer’s coding journey](https://github.blog/2022-06-09-introducing-achievements-recognizing-the-many-stages-of-a-developers-coding-journey/) by [Bailey Brooks](https://github.blog/author/bailey/) 

Thank you Github & Community.
![blog-header](https://user-images.githubusercontent.com/25535696/173086160-db4f3933-2b18-4bf1-9b0a-f3e9f905f2de.jpg)

**Comment by IvailoPetrov06** [comment]
You need to answer two discussions get Galaxy Brain badge:
To see more : https://github.com/Schweinepriester/github-profile-achievements

**Comment by JhonnyLusonode** [comment]
To get the Achivements Badges:

Pull Shark Badge:
 - You have to do branch and click pull request and merge it.

Galaxy Brain Badge:
 - When you answer on two discussions.
 
YOLO Badge:
 - Merged pull request without code review

Public Sponsor:
 - Sponsoring open source work via [GitHub Sponsors](https://github.com/sponsors)

If you need more details check:
https://github.com/Schweinepriester/github-profile-achievements
 
 If this reply is helpful, please kindly 'Mark as answer' so i can get the achievement too. Thanks!

**Comment by melvincwng** [comment]
@atulkamble Hi Atul, welcome to GitHub Community 👋🏻 

Happy achievement hunting :)
